### PR TITLE
Managing paired balance errors

### PR DESCRIPTION
**Pull request by banklesss**
Related to [YOMO-452](https://emurgo.atlassian.net/browse/YOMO-452)

Until now, when a request for the current price to get the exchange rate was made and failed, the error was displayed by hiding the balance and the exchange rate at the same time.

After this change, when a request for the current price is made and it fails a default paired balance inline error will be shown.

image

More info: https://emurgo.atlassian.net/browse/YOMO-452?focusedCommentId=14645

[YOMO-452]: https://emurgo.atlassian.net/browse/YOMO-452?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by stackchain — APPROVED** [review]
I'm ok with that.

**Comment by wolverineks** [comment]
why ignore the errors? why not wrap in a different boundary?

**Review by wolverineks — APPROVED** [review]

File: src/TxHistory/BalanceBanner.tsx [hunk 1]
```diff
@@ -41,8 +41,8 @@ export const BalanceBanner = () => {
             loading={{size: 'small'}}
             error={{
               size: 'inline',
-              fallback: () => (
-                <Text style={styles.pairedBalanceText} testID="pairedTotalText">
+              fallback: ({resetErrorBoundary}) => (
+                <Text onPress={resetErrorBoundary} style={styles.pairedBalanceText} testID="pairedTotalText">
```
Comment (wolverineks) [inline src/TxHistory/BalanceBanner.tsx]: use a button

**Review by wolverineks — CHANGES_REQUESTED** [review]
use a button

File: src/TxHistory/BalanceBanner.tsx (hunk 1 above)
Comment (stackchain) [reply src/TxHistory/BalanceBanner.tsx]: @jemubm check with the Design if we can add a refresh button at the end of the sentence (Icon.ArrowUpAndDown) or something alike.

Comment (banklesss) [reply src/TxHistory/BalanceBanner.tsx]: 👍🏻

**Comment by wolverineks** [comment]
keep it simple, get rid of all the ref stuff

**Comment by banklesss** [comment]
> keep it simple, get rid of all the ref stuff

Suggestion?

**Comment by wolverineks** [comment]
> > keep it simple, get rid of all the ref stuff
> 
> Suggestion?

for what?

**Comment by banklesss** [comment]
> > > keep it simple, get rid of all the ref stuff
> 
> > 
> 
> > Suggestion?
> 
> 
> 
> for what?

Could you provide an alternative way?

**Comment by wolverineks** [comment]
> > > > keep it simple, get rid of all the ref stuff
> > 
> > 
> > > 
> > 
> > 
> > > Suggestion?
> > 
> > 
> > for what?
> 
> Could you provide an alternative way?

to do what?

**Comment by banklesss** [comment]
> > > > > keep it simple, get rid of all the ref stuff
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > Suggestion?
> 
> > > 
> 
> > > 
> 
> > > for what?
> 
> > 
> 
> > Could you provide an alternative way?
> 
> 
> 
> to do what?

How would you recover the component from the boundary fallback on refresh?

**Review by stackchain — APPROVED** [review]

**Review by wolverineks — APPROVED** [review]

**Pull request merged**